lib/renderer/api/remote.js

```diff
@@ -72,7 +72,7 @@ function wrapArgs (args, visited = new Set()) {
         members: []
       }
       visited.add(value)
-      for (const prop in value) {
+      for (const prop in value) { // eslint-disable-line guard-for-in
```

Comment:
I think this could be `Object.keys`?

Reply:
Maybe, but I'm scared it will result in a side-effect breaking change.  We can flip it to `Object.keys` in a major version or when we extract `remote`